test/each-only.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit Main Test Suite</title>
```

Comment:
might be helpful to rename

Reply:
Updated here and `each.html` as well. The curse of copy-paste :-).